[DOCS] First pass at upgrade updates for 7.0. (#39944)
elastic/elasticsearch: docs/reference/upgrade/open-ml.asciidoc

POST _ml/set_upgrade_mode?enabled=false
--------------------------------------------------
// CONSOLE

> If you closed all {ml} jobs before the upgrade, open them and their associated

Reviewer: Not all jobs necessarily have datafeeds, so I'd suggest changing this to something like:

```suggestion
If you closed all {ml} jobs before the upgrade, open the jobs and start the
```

> datafeeds from {kib} or with the <<ml-open-job,open jobs API>>. 

Reviewer: ```suggestion
datafeeds from {kib} or with the <<ml-open-job,open jobs API>> and <<ml-start-datafeed,start datafeeds API>> . 
```
